.github/workflows/build.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
           - [ hc-scaffold ]
         include:
           - os: ubuntu-latest
-            template: holonix-default
```

Comment:
`default` is special, will it build with an empty string here? Then people can get the default template without specifying a template